sdks/python/apache_beam/io/gcp/pubsub.py

```diff
@@ -91,3 +140,13 @@ def display_data(self):
   def writer(self):
     raise NotImplementedError(
         'PubSubSink is not supported in local execution.')
+
+
+def _decodeUtf8String(encoded_value):
```

Comment:
These can be removed with coders passed into `ReadFromPubSub` and `WriteFromPubSub`.